openpype/hosts/blender/hooks/pre_add_run_python_script_arg.py

```diff
@@ -24,8 +24,8 @@ def execute(self):
             # Test script path exists
             python_script_path = Path(python_script_path)
             if not python_script_path.exists():
-                raise ValueError(
-                    f"Python script {python_script_path} doesn't exist."
+                raise self.log.warning(
```

Comment:
This doesn't look right, that 'raise' shouldn't be there, imho.

Reply:
Oh forgive me, I think I was tired...